ddtrace/tracer.py

```diff
@@ -80,13 +90,16 @@ def configure(self, enabled=None, hostname=None, port=None, sampler=None,
             Otherwise they'll be dropped.
         :param str hostname: Hostname running the Trace Agent
         :param int port: Port of the Trace Agent
-        :param object sampler: A custom Sampler instance
+        :param object sampler: A custom Sampler instance, locally deciding to totally drop the trace or not.
+        :param object priority_sampler: A custom Sampler instance, taking the priority sampling decision.
```

Comment:
Looks like my comment disappeared, but I still advocate for removing this from the `configure` API and only keep the `priority_sampling` flag.